### ISSUE TITLE
Getting blurry results after training

### ISSUE THREAD

**Issue by prasadxy** [NVIDIA/vid2vid#76]
Hi all,

I am trying to train vid2vid on pose dataset downloaded through `scripts/download_datasets.py` file. I have one GPU GTX 1080 card and hence changed few parameters in `scripts/pose/train_g1_256.sh` file. I have changed number of input channels to 3 and scales in spatial domain to 2. This is the script I am running:

```
python train.py --name pose2body_256p_g1 \
--dataroot datasets/pose --dataset_mode pose \
--input_nc 3 --ngf 64 --num_D 2 \
--resize_or_crop randomScaleHeight_and_scaledCrop --loadSize 384 --fineSize 256 \
--niter 5 --niter_decay 5 \
--no_first_img --n_frames_total 12 --max_frames_per_gpu 4 --max_t_step 4 \
--n_scales_spatial 2 --openpose_only 
```

After that, when I perform testing, I get very blurred images. I am attaching few of the images. 
![fake_b_frame001502](https://user-images.githubusercontent.com/45045347/48502909-b40f2600-e866-11e8-9a54-332ad0261bd4.jpg)
![fake_b_frame001503](https://user-images.githubusercontent.com/45045347/48502911-b40f2600-e866-11e8-8bd0-f82774ec46bb.jpg)
![fake_b_frame001504](https://user-images.githubusercontent.com/45045347/48502913-b4a7bc80-e866-11e8-80d6-53449d332699.jpg)
![fake_b_frame001505](https://user-images.githubusercontent.com/45045347/48502915-b4a7bc80-e866-11e8-90b3-5e2c38a3e7fa.jpg)

Please can somebody explain how to get proper results? Thanks in advance.

**Comment by nfrik** [comment]
@dimaxano could you please share more about why openpose+densepose would cause blurriness? I would imagine that adding openpose data should not decrease the quality of the model, but unfortunately I didn't have much improvement in model quality when I use openpose+densepose.

**Comment by dimaxano** [comment]
You are right, openpose+densepose should get better results. But in your command I see, that you are using openpose data only. So it may be the cause of your problem

**Comment by OptimusPrimeCao** [comment]
If using densepose(points enough to represent the person's figure) to train the model, how to deal with the discrepancy between figure 
of source person and that of target person in  test time?
